app/editor/src/hooks/modal/useModal.ts

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const useModal = () => {
```

Comment:
Not certain of the value of this hook.  It's not currently connected to the Modal in anyway.

Reply:
Was just to cut down some time as it will be the same on every modal, can remove though